src/gui/rss/rssparser.cpp

```diff
@@ -266,6 +272,11 @@ void RssParser::parseRssArticle(QXmlStreamReader& xml, const QString& feedUrl)
         else
           article["news_link"] = link;
       }
+      else if(xml.name() == "magnetURI") {
```

Comment:
The RSS feed you shared has the torrent file and the magnet link. In some situations your code will prefer to use the magnet instead of the torrent. We always prefer the torrent @sledgehammer999 ?